src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php

```diff
@@ -85,8 +88,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     return $disabled === true ? ['disabled' => $disabled] : [];
                 },
                 'expanded' => true,
-                'required' => false,
                 'placeholder' => false,
+                'row_attr' => [
```

Comment:
~~this option doesn't exists in Symfony 3, is it something PrestaShop have backported into its own form theme ?~~ ok  :)